templates/pages/reset-password.jsx

```diff
@@ -53,6 +55,7 @@ var ResetPassword = React.createClass({
   },
   handleRequestPassword: function(error, data) {
     if ( error ) {
+      ga.event({category: 'Reset Password', action: 'Error form validation on request password page'});
```

Comment:
Also ignore this comment - as the specific field validation is captured by form.jsx